HW9/task2.py

```diff
@@ -0,0 +1,35 @@
+class Employee:
+    def __init__(self, name: str, genre: str = None, age: int = None, location: str = None, birth: str = None,
+                 specialization: str = None, salary: int = None, working_years: float = 0.0):
+        self.name = name
+        self.genre = genre
+        self.age = age
+        self.location = location
+        self.birth = birth
+        self.specialization = specialization
+        self.salary = salary
+        self.working_years = working_years
```

Comment:
why are all these vars public? 

Where are properties with limitations for them?